packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -228,6 +231,30 @@ class _SelectableRegionState extends State<SelectableRegion> with TextSelectionD
     );
   }
 
+  @override
+  void didChangeDependencies() {
+    super.didChangeDependencies();
+    if (defaultTargetPlatform != TargetPlatform.iOS && defaultTargetPlatform != TargetPlatform.android) {
```

Comment:
drive by comment: here and elsewhere use a `switch` for enums. This helps us track down places to update if we happen to add more platforms in the future 😅